frontend/packages/ceph-storage-plugin/src/components/dashboards/persistent-internal/status-card/status-card.tsx

```diff
@@ -74,6 +127,12 @@ export const StatusCard: React.FC<DashboardItemProps> = ({
     t,
   );
 
+  const [isHCPresent, setHCPresence] = React.useState(false);
+  React.useEffect(() => {
+    const cephDetails = data?.[0]?.status?.ceph?.details;
+    setHCPresence(!!cephDetails);
+  }, [data]);
+
```

Comment:
It can be a value. No good reason to persist as state.
```suggestion
const isHCPresent = data?.[0]?.status?.ceph?.details;
```